src/magic_folder/test/strategies.py

```diff
@@ -317,3 +344,35 @@ def unique_value_dictionaries(keys, values, min_size=None, max_size=None):
             lambda values: dict(zip(keys, values)),
         ),
     )
+
+
+def remote_snapshots(names=path_segments(), authors=remote_authors()):
+    """
+    Build ``RemoteSnapshot`` instances.
+    """
+    return builds(
+        RemoteSnapshot,
+        name=names,
+        author=authors,
+        metadata=dictionaries(text(), text()),
+        capability=tahoe_lafs_dir_capabilities(),
+        parents_raw=lists(tahoe_lafs_dir_capabilities()),
```

Comment:
Not sure if this is that big a deal, but `tahoe_lafs_dir_capabilities()` will look like mutable directories -- but all snapshots will be immutable directories so the parents list should all contain immutable-caps.

The `capability` should similarly be an immutable-directory.

Reply:
Nice catch.  Yea, it would be better for these values to be realistic.